source/diagrams/manage-diagrams/create-diagrams.txt

```diff
@@ -10,4 +10,46 @@ Create Diagrams
    :depth: 1
    :class: singlecol
 
-Create diagrams.
\ No newline at end of file
+You can create new diagrams on the Mapping tab of Relational Migrator.
+You can also create multiple diagrams and navigate between them to view
+specific entities of your data model. By default, Relational Migrator
+creates the ``Main Diagram`` diagram with a Relational view and a
+MongoDB view.
+
+About this Task
+---------------
+
+Creating diagrams allows you to visualize tables, foreign key
```

Comment:
**Suggestion**

Convert these items under AtT to bullet points.

Reply:
Done! Thanks for your feedback on this one!